RELEASE-NOTES.rst

```diff
@@ -18,6 +18,226 @@
+~~~~~~~~~~~~~
+
+-  Updates installation procedure with instruction to set the correct
+   version of ``pipenv`` and ``python``.
+-  Adds a flask ``Flask-Wiki`` module to display and edit help
+   documentation for the end users.
+-  Updates the contributors list of the ``rero-ils-ui`` project.
+
+Issues
+~~~~~~
+
+-  `rero-ils-ui#169 <https://github.com/rero/rero-ils-ui/issue/169>`__:
+   A CSS styling rule was missing on the person detailed view of the
+   professional interface to reduce the size of the source information
+   badges.
+-  `rero-ils-ui#209 <https://github.com/rero/rero-ils-ui/issue/209>`__:
+   In the patron account fee tab of the professional interface, the
+   actions drop down menu was not placed just below the action button.
+-  `#538 <https://github.com/rero/rero-ils/issue/538>`__: Help messages
+   (ie JSON schema description fields or validation messages) were
+   missing in the patron editor.
+-  `#575 <https://github.com/rero/rero-ils/issue/575>`__: The library
+   editor was accessible to any librarian typing the correct URL in the
+   web browser. The record couldn’t be saved, but still.
+-  `#787 <https://github.com/rero/rero-ils/issues/787>`__: As a
+   generalization of issue
+   `#575 <https://github.com/rero/rero-ils/issue/575>`__, access to
+   resources editor had to be protected based on the user permissions.
+-  `#793 <https://github.com/rero/rero-ils/issues/793>`__: In some
+   cases, the patron displayed in the checkin interface wasn’t correct.
+   To solved this, better information had to be returned after the
+   checkin.
+-  `#794 <https://github.com/rero/rero-ils/issues/794>`__: The
+   ``pickup_name`` field of a location that is pickup wasn’t required,
+   thus resulting in incomplete records when creating or updating a
+   location through the editor.
+-  `#798 <https://github.com/rero/rero-ils/issues/798>`__: The
+   professional item detailed view didn’t display information on why an
+   item isn’t available in some cases.
+-  `#803 <https://github.com/rero/rero-ils/issue/803>`__: In the request
+   validation interface, when the librarian validated a request, the
+   focus form the input field was lost, forcing the librarian to click
+   to set the focus for the next validation.
+-  `#804 <https://github.com/rero/rero-ils/issue/804>`__: Example had to
+   be added in the patron editor to help the end user.
+-  `#826 <https://github.com/rero/rero-ils/issue/826>`__: In the checkin
+   interface, when a checkin item has a request, the name of the patron
+   that placed the request was not displayed in the correct order (last
+   name, first name).
+-  `#827 <https://github.com/rero/rero-ils/issue/827>`__: The alignment
+   of component had to be improved in the circulation interface. Items
+   with an action button were shorter than items without any button.
+-  `#829 <https://github.com/rero/rero-ils/issue/829>`__: Some flash
+   messages were missing when the librarian in checkin in items that has
```

Comment:
*when the librarian in checkin in items that has requests* [something's wrong here...]

Reply:
Yes, it should be "when the librarian **is** checkin in items that have requests".